src/Features/LanguageServer/Protocol/Handler/SpellCheck/AbstractSpellCheckingHandler.cs

```diff
@@ -5,17 +5,16 @@
 using System;
 using System.Collections.Generic;
 using System.Collections.Immutable;
+using System.Runtime.CompilerServices;
+using System.Security.Cryptography;
```

Comment:
what is using this?

Reply:
nothing.  must have been an errant add from completion :)